pkg/governance/tool.go

```diff
@@ -0,0 +1,44 @@
+package governance
+
+import (
+	"fmt"
+	"github.com/harmony-one/harmony/accounts"
+	"github.com/harmony-one/harmony/accounts/keystore"
+	"github.com/harmony-one/harmony/crypto/hash"
+	"strconv"
+	"strings"
+	"time"
+)
+
+func timestampToDateString(timestamp float64) string {
+	return time.Unix(int64(timestamp), 0).Format(time.RFC822)
+}
+
+func linePaddingPrint(content string, trim bool) {
+	for _, line := range strings.Split(content, "\n") {
+		trimLine := line
+		if trim {
+			trimLine = strings.TrimSpace(line)
+		}
+		if trimLine != "" {
+			fmt.Printf("        %s\n", trimLine)
+		}
+	}
+}
+
+func signMessage(keyStore *keystore.KeyStore, account accounts.Account, data []byte) (sign []byte, err error) {
+	signData := append([]byte("\x19Ethereum Signed Message:\n" + strconv.Itoa(len(data))))
```

Comment:
Does it has to be "Ethereum Signed Message:"?  Can you change it to "Harmony Signed Message:"?

Reply:
It has to be "Ethereum Signed Message:", beacause the backend and onewallet also use it.